tests/test_workflow/test_methode_attribute.py

```diff
@@ -0,0 +1,148 @@
+from dessia_common import tests
+import unittest
+from dessia_common import core
+from dessia_common.workflow.blocks import GetModelAttribute, SetModelAttribute
+from dessia_common.workflow.core import Variable, TypedVariable
+import inspect
+from dessia_common.typings import AttributeType
+from dessia_common.forms import DessiaObject
+import time
+from typing import List, Dict
+
+class UnrealClassForTesting(DessiaObject):
```

Comment:
A better name would be `DummyClassForTesting`

Reply:
Done